src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/SphKerrSchild.cpp

```diff
@@ -0,0 +1,1053 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/SphKerrSchild.hpp"
+
+#include <cmath>  // IWYU pragma: keep
+#include <numeric>
+#include <ostream>
+#include <typeinfo>
+#include <utility>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/EagerMath/CrossProduct.hpp"
+#include "DataStructures/Tensor/EagerMath/Determinant.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ExtrinsicCurvature.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/StdArrayHelpers.hpp"
+#include "Utilities/StdHelpers.hpp"
+
+namespace gr::Solutions {
+
+SphKerrSchild::SphKerrSchild(const double mass,
+                             SphKerrSchild::Spin::type dimensionless_spin,
+                             SphKerrSchild::Center::type center,
+                             const Options::Context& context)
+
+    : mass_(mass),
+      // clang-tidy: do not std::move trivial types.
```

Comment:
Nowadays we prefer `// NOLINTNEXTLINE(...)` You can figure out the `...` by removing the `NOLINT` and seeing what error clang-tidy gives you 🙃